HtmlMesh/src/html-mesh.ts

```diff
@@ -165,16 +165,16 @@ export class HtmlMesh extends Mesh {
      * @param {number} height
      */
     setContentSizePx(width: number, height: number) {
-        if (!this._element || !this._wrapElement) {
+        if (!this._element || !this._sizingElement) {
             return;
         }
-        const contentElement = this._wrapElement.firstElementChild! as HTMLElement;
+        const contentElement = this._sizingElement.firstElementChild! as HTMLElement;
         contentElement.style.transform = "none";
         // const childElement = contentElement;
         const [childWidth, childHeight] = [contentElement.offsetWidth, contentElement.offsetHeight];
         if (contentElement) {
-            this._wrapElement.style.width = `${width}px`;
-            this._wrapElement.style.height = `${height}px`;
+            this._sizingElement.style.width = `${width}px`;
+            this._sizingElement.style.height = `${height}px`;
             const transform = `scale(${Math.min(width / childWidth, height / childHeight)})`;
```

Comment:
@CTOSPACE I know we already merged this PR, but could you explain what problem we were solving by introducing the sizing element?  As I am investigating the camera rotation issue, I am noticing that it is introducing a scale up transform, which we want to avoid as it will make the text less clear.  This is why I size the element to fill the screen and then scale it down to fit the mesh.  This ensures that is we zoom all the way in, we are never upscaling so we get the clearest text.  We may need to find another way to solve whatever issue you were seeing, that doesn't involve an upscale.

Reply:
My idea is to want the HtmlMesh element to be like the background image of an HTML element, able to adjust its size and position based on the size of the parent element and its own size. You can imagine the HtmlMesh element as the background image of a certain HTML element, and we can use background size to control its size, such as contain, cover, or stretch. Currently, my current implementation only focuses on contain and center, If the font becomes blurry, it is necessary to find the reason, because it is set through scale. HTML text is vector scaled for scale, and it will not become blurry due to a large scale, right

Reply:
HTML text is not vector scaled.  When you scale an HTML element, the browser increases the size of the rendered pixels, which can lead to blurriness or pixelation if the scale factor is large.  You may be thinking of modern browsers using subpixel rendering and font hinting to try to make the text appear sharper, but they can't completely eliminate the bluriness.  I've seen this in practice, typically when zooming in, thereby increasing the scale.  In what way did you find the HtmlMesh was unable to adjust its position or size?  If you scale the HtmlMesh (directly or via its parent), it will scale the element to keep it the same size.  If you adjust the size of the element, I suspect it will not, because we would need to recompute the base scale  factor.  We could add a resize listener to handle that case, if it is important.  Having said that, I did use a similar wrapper concept that I added outside the mesh in the example to handle small discrepancies in size (it is impossible to do the downscaling approach and get the size to be perfect).  I like that you added it as a default, though I could imagine the user may want to adjust the color of the wrapper to be other than black.  So if the primary purpose of the wrapper was to ensure that the entire mesh is filled with HTML content (either the content element or the wrapper) I see what you were trying to achieve and that can be done without introducing upscaling.  I'll push a change.